src/pages/SendConfirm.vue

```diff
@@ -0,0 +1,322 @@
+    <guide
+      fill="neutral"
+      icon="³⁄₃"
+    >
+      <em>Review your transfer</em>
+      <br>from
+      <ae-identicon
+        :address="activeIdentity.address"
+        size="s"
+      /><em>{{ activeIdentity.name }}</em>
+      <br>to
+      <ae-identicon
+        :address="$route.params.to"
+        size="s"
+      />
+      <em>
+        <span class="address">
+          {{ formattedAddress }}
+        </span>
+      </em>
+    </guide>
+
+    <list-item inactive>
+      <span class="title">
+        Amount
+      </span>
+      <span
+        slot="right"
+        class="content"
+      >
+        {{ BigNumber($route.params.amount) | prefixedAmount }} AE
+      </span>
+    </list-item>
+    <list-item inactive>
+      <span class="title">
+        <img src="../assets/icons/turtle.svg">
+      </span>
+      <span
+        slot="right"
+        class="content"
+      >
+        <img src="../assets/icons/rabbit.svg">
+      </span>
+    </list-item>
+    <list-item inactive>
+      <input
+        v-model="fee"
+        :min="MIN_SPEND_TX_FEE"
+        :max="MAX_REASONABLE_FEE"
+        type="range"
+      >
+    </list-item>
+    <list-item inactive>
+      <div class="title">
+        Gas Price
+      </div>
+      <span
+        slot="right"
+        class="content"
+      >
+        {{ formattedFee | prefixedAmount }} AE
+      </span>
+    </list-item>
+    <list-item inactive>
+      <div class="title">
+        Transaction speed
+      </div>
+      <span
+        slot="right"
+        class="content"
+      >
+        {{ transactionSpeed }}
+      </span>
+    </list-item>
+    <list-item
+      inactive
+      class="account"
+    >
+      <div class="title">
+        Recipient Account Key
+      </div>
+      <div class="subtitle">
+        <ae-address :value="$route.params.to" />
+      </div>
+    </list-item>
+
+    <ae-button-group slot="footer">
+      <ae-button
+        :to="{ name: 'transfer' }"
+        uppercase
+      >
+        Cancel
+      </ae-button>
+      <ae-button
+        fill="secondary"
+        uppercase
+        @click="send"
+      >
+        Confirm
+      </ae-button>
+    </ae-button-group>
+  </mobile-page>
+</template>
+
+<script>
+import BigNumber from 'bignumber.js';
+import { mapGetters, mapState } from 'vuex';
+import { AeIdenticon, AeButtonGroup, AeAddress } from '@aeternity/aepp-components-3';
+import MobilePage from '../components/MobilePage.vue';
+import Guide from '../components/Guide.vue';
+import ListItem from '../components/deprecated/ListItem.vue';
+import AeButton from '../components/AeButton.vue';
+import { MAGNITUDE, MIN_SPEND_TX_FEE, MAX_REASONABLE_FEE } from '../lib/constants';
+import prefixedAmount from '../filters/prefixedAmount';
+
+export default {
+  components: {
+    MobilePage,
+    Guide,
+    AeIdenticon,
+    ListItem,
+    AeButton,
+    AeButtonGroup,
+    AeAddress,
+  },
+  filters: { prefixedAmount },
+  data: () => ({
+    amount: '',
+    MAGNITUDE,
```

Comment:
this is not used in the template, so let's use just `MAGNITUDE` instead of `this.MAGNITUDE` below